pkgs/development/python-modules/libsass/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildPythonPackage, fetchFromGitHub, libsass, six }:
+
+buildPythonPackage rec {
+  pname = "libsass";
+  version = "0.20.1";
+
+  src = fetchFromGitHub {
+    owner = "sass";
+    repo = "libsass-python";
+    rev = version;
+    sha256 = "1r0kgl7i6nnhgjl44sjw57k08gh2qr7l8slqih550dyxbf1akbxh";
+  };
+
+  buildInputs = [
+    libsass
+  ];
+
+  propagatedBuildInputs = [
+    six
+  ];
+
+  preBuild = ''
+    export SYSTEM_SASS=true;
+  '';
+
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```